packages/linear-progress/src/lib/linear-progress.tsx

```diff
@@ -0,0 +1,34 @@
+import MuiLinearProgress, {
+  LinearProgressProps as MuiLinearProgressProps,
+} from '@mui/material/LinearProgress';
+import Box from '@mui/material/Box';
+import { styled } from '@novatics/styles';
+import { Typography } from '@mui/material';
+
+interface LinearProgressProps extends MuiLinearProgressProps {
+  withLabel?: boolean;
```

Comment:
maybe showProgress or percentage?  withLabel gives me impression of something else

Reply:
I like "showPercentage", makes sense to me